### ISSUE TITLE
[Question] What does the --reset option do?

### ISSUE THREAD

**Issue by 1player** [flathub/com.valvesoftware.Steam#930]
Sorry, I don't see a way of opening a discussion without creating an issue.

The wiki says to run `flatpak run com.valvesoftware.Steam --reset` if there's issues with one's flatpak installation but doesn't say what this does. Does this clear `~/.var/app/com.valvesoftware.Steam`? Does it delete all game folders and their saves?

I think it's worth adding a line to the wiki to explain this.

**Comment by nanonyme** [comment]
It should potentially link to upstream wiki. What it does is it will call upstream executable with parameter --reset

**Comment by 1player** [comment]
Ah, thanks. I can't find any information online, but I manage to track down its behaviour in the `reset_steam` function in `/.var/app/com.valvesoftware.Steam/.local/share/Steam/steam.sh`.

tl;dr: removes any steam file and binary, but keeps game data and saves intact. I'll update the wiki.

**Comment by 1player** [comment]
Well, I'll reopen and turn this into a bug because `flatpak run com.valvesoftware.Steam --reset` doesn't seem to do anything on my machine.

It just starts Steam normally, without running that `reset_steam` function and cleaning any files.

**Comment by lainedfles** [comment]
Hi, this works for me. Note `--command=.local/share/Steam/steam.sh`:

```bash
/usr/bin/flatpak run --branch=stable --arch=x86_64 --command=.local/share/Steam/steam.sh --file-forwarding com.valvesoftware.Steam --reset
```
